Feature/sprint 2

tgcgowtham opened the pull request:
Merging Sprint 2 changes to Main Branch.

Commit:
Added logger conds

Commit:
Pushing local dev changes, added model notebooks & class files

Commit:
Added modelWrapper

Commit:
Added notebook

Commit:
Removed temp images

Commit:
Ext folder created

Commit:
Removed temp images

kilarimohith7 committed:
added data generator class for issue 2

Iftequ committed:
Added Model file

Added a notebook and check the functionality of the model data.

Commit:
Added testcase file

Commit:
Added Models

bsr120602 committed:
Merge pull request #11 from gowthamchanduthota/feature/issue-2-pull-images-to-datagen

Feature/issue 2 pull images to datagen

Commit:
Updated svae model fn

kilarimohith7 committed:
Merge pull request #12 from gowthamchanduthota/feature/issue-5-datagenerator-class

Feature/issue 5 datagenerator class

Commit:
Added logging in DataGenerator

Iftequ commented on sprint-2/DataGenerator.py:
Do we have Potential issues with bias or overfitting?

kilarimohith7 committed:
Update s3helper.py

kilarimohith7 committed:
Update s3helper.py

Used os.path.basename for a cleaner way to get the filename.
Updated the get_img function to specify color mode more concisely based on greyscale.
Enhanced the upload functionality by adding s3_key formatting in main for a timestamped upload path.

kilarimohith7 committed:
Update script.py

Added logging for key events, error handling for file download/upload, and a loop to upload randomly sampled files to S3 with error tracking.

kilarimohith7 committed:
Update ModelWrapper.py

Added logging functionality to track operations.

kilarimohith7 committed:
Update DataGenerator.py

Added logging for batch processing and a simple data augmentation step to the __data_generation method.

bsr120602 committed:
Update _newmodel.ipynb

Change the a part of the code for improving the efficiency

bsr120602 committed:
Update _newmodel.ipynb

changed a small part of the code for improving the efficiency

bsr120602 committed:
Update _newmodel2.ipynb

changed a function little bit

bsr120602 committed:
Update _newmodel.ipynb

bsr120602 committed:
Update _newmodel.ipynb

Added some new lines of code in place of code

bsr120602 committed:
Update _newmodel3.ipynb

added few lines in data frame

bsr120602 committed:
Update _newmodel.ipynb

bsr120602 committed:
Update _newmodel.ipynb

Added a few lines of code

Iftequ committed:
Update oldmodel.ipynb

Also check the library functions.

Iftequ committed:
Update _newmodel2.ipynb

check the vulnerabilities in the code

Iftequ committed:
Update README.md

scikit-learn-aws-s3-extension

Iftequ committed:
Update test.py

Iftequ committed:
Update Advertising.csv

Iftequ committed:
Update _newmodel2.ipynb

Iftequ committed:
Update tests.py

bsr120602 committed:
Update DataGenerator.py

Consistent d type initialization , enhancing the batch processing , and streamlined for easier to understand

bsr120602 committed:
Update tests.py

Optimized label extraction , setup refactoring , moved model build creation to a seperate build_ model function for modularity , improved assertions

bsr120602 committed:
Update _newmodel3.ipynb

Used.np.random.permutation , used zip for simultaneous unpacking , improved consistency with naming conventions for lists

bsr120602 committed:
Update _newmodel3.ipynb

bsr120602 committed:
Update test.py

Fixed lists to llist_objects and changed the code for best optimal solution
